frontend/src/app/lib/InstallManager.js

```diff
@@ -215,9 +201,8 @@ export function enableExperiment(dispatch, experiment, sendToGA) {
       inProgress: true
     })
   );
-
-  return mam
-    .getAddonByID(experiment.addon_id)
+  return installAddon(sendToGA, eventCategory, eventLabel, experiment.slug)
```

Comment:
here's the part that ensures the txp addon is installed when an experiment is enabled